src/components/entities/listItems/channel.js

```diff
@@ -151,10 +151,9 @@ const Channel = (props: Props) => {
           <Tooltip content={tipText}>
             <span style={{ marginLeft: '8px', display: 'flex' }}>
               <OutlineButton
-                disabled={isLoading}
```

Comment:
If we remove this, the user could double click and we might end up with some race conditions on network requests, right? Is there any way to preserve this?

Reply:
🤔  I decided to remove the `disabled` prop because in reality it is not preventing the double click right now. The component with the `onClick` functionality is `ToggleChannelNotifications` that is the parent of the button. The `disabled` at the moment is only giving the opacity effect.

**Current app functionality**

![Jul-22-2020 13-43-54](https://user-images.githubusercontent.com/1315054/88209478-c71fef00-cc20-11ea-9c27-6af6efd95141.gif)

Reply:
I could also prevent that in this PR if you want.